daprdocs/content/en/developing-applications/building-blocks/configuration/howto-manage-configuration.md

```diff
@@ -1,32 +1,26 @@
 ---
 type: docs
-title: "How-To: Manage application configuration"
-linkTitle: "How-To: Manage application configuration"
+title: "How-To: Manage configuration from a store"
+linkTitle: "How-To: Manage configuration from a store"
 weight: 2000
-description: "Learn how to get application configuration and watch for changes"
+description: "Learn how to get application configuration and subscribe for changes"
 ---
 
 ## Introduction
+This HowTo uses the Redis configuration store component as an example on how to retrieve a configuration item.
 
-Consuming application configuration is a common task when writing applications and frequently configuration stores are used to manage this configuration data. A configuration item is often dynamic in nature and is tightly coupled to the needs of the application that consumes it. For example, common uses for application configuration include names of secrets that need to be retrieved, different identifiers, partition or consumer IDs, names of databased to connect to etc. These configuration items are typically stored as key-value items in a database.
+*This API is currently in `Alpha state` and only available on gRPC. An HTTP1.1 supported version with this URL `/v1.0/configuration` will be available before the API becomes stable.*
 
-Dapr provides a [State Management API]({{<ref "state-management-overview.md">}})) that is based on key-value stores. However, application configuration can be changed by either developers or operators at runtime and the developer needs to be notified of these changes in order to take the required action and load the new configuration. Also the configuration data may want to be read only. Dapr's Configuration API allows developers to consume configuration items that are returned as key/value pair and subscribe to changes whenever a configuration item changes.
-
-*This API is currently in `Alpha state` and only available on gRPC. An HTTP1.1 supported version with this URL `/v1.0/configuration` will be available before the API becomes stable. *
-
-This HowTo uses the Redis configuration store component as an exmaple to retrieve a configuration item.
-
-## Step 1: Save a configuration item
+## Step 1: Create a configuration item in store
 
 First, create a configuration item in a supported configuration store. This can be a simple key-value item, with any key of your choice. For this example, we'll use the Redis configuration store component
```

Comment:
```suggestion
First, create a configuration item in a supported configuration store. This can be a simple key-value item, with any key of your choice. For this example, we'll use the Redis configuration store component.
```